lms/static/certificates/sass/_print.scss

```diff
@@ -94,6 +94,12 @@
 // ------------------------------
 @media print {
 
+    html,
+    body {
+        height: 650px;
```

Comment:
Not really finding anything JavaScript-wise either. The only thing that I can think of would be to generate a PDF on-the-fly, but that defeats the purpose of our web view.